### PR TITLE
perf(aad-manifest): partially disable aad manifest json intellisense

### PR DESCRIPTION
**Pull request by SLdragon**
Fix this bug: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/AuthAndData/Microsoft%20Teams%20Extensibility/CY22-4.2?workitem=14171623

Due to limitation of VSCode, we can only partially disable intellisense using empty JSON schema, see this issue for details: https://github.com/microsoft/vscode/issues/63270

Use empty JSON schema in VSCode settings.json file:
```
{
    "json.schemas": [
        {
            "fileMatch": [
                "/aad.*.json"
            ],
            "schema": {}
        }
    ]
}
```

### CODE REVIEW

**Review by KennethBWSong — DISMISSED** [review]

**Review by swatDong — DISMISSED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts [hunk 1]
```diff
@@ -115,15 +115,6 @@ export async function _scaffoldLocalDebugSettings(
             EOL: os.EOL,
           }
         );
-
-        await fs.writeJSON(
```
Comment (blackchoey) [inline packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts:119]: seems we only updated local debug's logic. is there any other logic that generates vscode settings?

**Review by blackchoey — APPROVED** [review]

**Comment by blackchoey** [comment]
By the way, can we add a UT for this change?

File: packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts:119]: Nope, the generate vscode settings.json file only exist in local debug scaffold stage

**Comment by SLdragon** [comment]
> By the way, can we add a UT for this change?

Thanks, will update UT in this PR

**Review by swatDong — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Review by blackchoey — APPROVED** [review]

**Pull request merged**